velox/vector/arrow/tests/ArrowBridgeArrayTest.cpp

```diff
@@ -1135,38 +1135,45 @@ class ArrowBridgeArrayImportTest : public ArrowBridgeArrayExportTest {
         "ttn", {Timestamp(0, 0), std::nullopt, Timestamp(1699308257, 1234)});
   }
 
-  void testImportWithoutNullsBuffer() {
-    std::vector<std::optional<int64_t>> inputValues = {1, 2, 3, 4, 5};
+  template <typename TInput, typename TOutput>
```

Comment:
nit: `TInput` can be inferred from `inputValues` so we can move it to end of the type parameter list and omit it at call sites.

Reply:
Thanks for your suggestion. Updated.